### PR TITLE
Add blast button and textbox to UI

### PR DESCRIPTION
**Pull request by krupalip**
https://github.com/BioplatformsAustralia/bpaotu/issues/99

### CODE REVIEW

File: bpaotu/bpaotu/frontend/src/pages/search_page/reducers/types.ts [hunk 1]
```diff
@@ -24,6 +24,13 @@ export interface GalaxySubmission {
   error?: string
 }
 
+export interface BlastSubmission {
+  submissionId?: string
```
Comment (sztamas) [inline bpaotu/bpaotu/frontend/src/pages/search_page/reducers/types.ts]: Why is this optional? Don't we always have a submissionId?

File: bpaotu/bpaotu/frontend/src/pages/search_page/reducers/blast_search.ts [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        const alerts = [BLAST_ALERT_IN_PROGRESS]
+
+        return {
+          ...state,
+          sequenceValue: state.sequenceValue,
+          alerts,
+          submissions: [...state.submissions, lastSubmission]
+        }
+      },
+      throw: (state, action: any) => ({
```
Comment (sztamas) [inline bpaotu/bpaotu/frontend/src/pages/search_page/reducers/blast_search.ts]: You probably want to display some error message here?

Comment (krupalip) [reply bpaotu/bpaotu/frontend/src/pages/search_page/reducers/blast_search.ts]: error alert added

File: bpaotu/bpaotu/frontend/src/pages/search_page/reducers/types.ts (hunk 1 above)
Comment (krupalip) [reply bpaotu/bpaotu/frontend/src/pages/search_page/reducers/types.ts]: fixed

**Pull request merged**